pallets/loans/src/lib.rs

```diff
@@ -1071,7 +1071,10 @@ pub mod pallet {
 			let rates = T::InterestAccrual::rates();
 			let prices = match input_prices {
 				PriceCollectionInput::Empty => BTreeMap::default(),
-				PriceCollectionInput::Custom(prices) => prices.into(),
+				PriceCollectionInput::Custom(prices) => {
```

Comment:
Could make sense to receive the time from the input?
```rust
PriceCollectionInput::Custom(prices, now); //In the oracle denomination
```

Reply:
I think it's the less invasive change, and also allows us to emulate old prices

Reply:
mmh, on a second thought, might makes sense.